docs/user-manuals/advancedstatefulset.md

```diff
@@ -142,6 +142,27 @@ Then controller will delete Pod-3 and create Pod-5 (existing Pods will be `[0,2,
 - If you just want to delete Pod-3, you should append `3` into `reserveOrdinals` list and set `replicas` to `3`.
 Then controller will delete Pod-3 (existing Pods will be `[0,2,4]`).
 
+### Specify Pod Deletion
+
+**FEATURE STATE:** Kruise v1.5.5, Kruise v1.6.4, Kruise v1.7.2
+
+Compared to manually deleting a Pod directly, using the `apps.kruise.io/specified-delete: true` specified Pod deletion will be protected by the `maxUnavailable` of the Advanced StatefulSet during deletion, and it will trigger the `PreparingDelete` lifecycle hook (see below).
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    # ...
+    apps.kruise.io/specified-delete: true
+spec:
+  containers:
+    - name: main
+    # ...
+```
+
+When the controller receives the above Pod update, it will prioritize the deletion process of Pods with the specified deletion label and ensure that the `maxUnavailable` limit is not exceeded.
```

Comment:
it will prioritize the deletion process of Pods with the specified deletion label 
-> it will trigger the deletion process of the pod with specified deletion label . 

plz also comment that the pod will be rebuild by the workload if the replicas is not matched after deletion.

Reply:
I emphasize prioritizing deletion over updates because I want to highlight that deletion takes precedence.


I think "the pod will be rebuilt by the workload if the replicas are not matched after deletion" should perhaps be "The pod will be rebuilt by the workload if the ordinal is not reserved." 

This is because even if the replicas do not match after deletion, the pod will still be rebuilt, and the pod with the highest ordinal will be deleted.